### PR TITLE
Release/v0.74.10 fix.1

### PR DESCRIPTION
**Pull request by EVODelavega**
DO NOT MERGE

```
{
  "system-tests": "release/v0.74.2",
  "vegacapsule": "v0.74.1"
}
```

### CODE REVIEW

File: core/monitor/price/snapshot.go [hunk 1]
```diff
@@ -174,15 +174,15 @@ func (e *Engine) serialisePriceRanges() []*types.PriceRangeCache {
 		})
 	}
 
-	sort.Slice(prc, func(i, j int) bool {
+	sort.SliceStable(prc, func(i, j int) bool {
 		if prc[i].Bound.UpFactor.Equal(prc[j].Bound.UpFactor) {
 			if prc[i].Bound.DownFactor.Equal(prc[j].Bound.DownFactor) {
-				return prc[i].Bound.Trigger.Horizon < prc[j].Bound.Trigger.Horizon
+				return prc[i].Bound.Trigger.Horizon < prc[j].Bound.Trigger.Horizon && prc[i].Bound.Active == prc[j].Bound.Active
```
Comment (ze97286) [inline core/monitor/price/snapshot.go:180]: that doesn't seem correct. shouldn't it be:

``
if prc[i].Bound.Trigger.Horizon == prc[j].Bound.Trigger.Horizon {
        return prc[i].Bound.Active == prc[j].Bound.Active
} 
return  prc[i].Bound.Trigger.Horizon < prc[j].Bound.Trigger.Horizon

File: core/monitor/price/snapshot.go [hunk 2]
```diff
@@ -177,12 +177,15 @@ func (e *Engine) serialisePriceRanges() []*types.PriceRangeCache {
...
+					return prc[i].Bound.Active == prc[j].Bound.Active
+				}
+				return prc[i].Bound.Trigger.Horizon < prc[j].Bound.Trigger.Horizon
 			}
-			return prc[j].Bound.DownFactor.LessThan(prc[i].Bound.DownFactor) && prc[i].Bound.Active == prc[j].Bound.Active
+			return prc[j].Bound.DownFactor.LessThan(prc[i].Bound.DownFactor)
 		}
 
-		return prc[j].Bound.UpFactor.GreaterThan(prc[i].Bound.UpFactor) && prc[i].Bound.Active == prc[j].Bound.Active
+		return prc[j].Bound.UpFactor.GreaterThan(prc[i].Bound.UpFactor)
```
Comment (ze97286) [inline core/monitor/price/snapshot.go:188]: not really important but why reverse sort rather than return LessThan like everywhere?

Comment (EVODelavega) [reply core/monitor/price/snapshot.go:188]: IDK, this is what was there

Comment (ze97286) [reply core/monitor/price/snapshot.go:188]: so was the bug :)

File: core/monitor/price/snapshot.go [hunk 3]
```diff
@@ -176,11 +176,11 @@ func (e *Engine) SerialisePriceRanges() []*types.PriceRangeCache {
 	}
 
 	sort.SliceStable(prc, func(i, j int) bool {
+		if prc[i].Bound.Active != prc[j].Bound.Active {
```
[the remark this answers is not in the document]
Comment (EVODelavega) [reply core/monitor/price/snapshot.go:179]: Inactive bounds move to the end of the slice, multiple inactive bounds will be sorted in the same way active bounds are

**Pull request closed without merging**